app/uk/gov/hmrc/excisemovementcontrolsystemapi/config/AppConfig.scala

```diff
@@ -16,25 +16,38 @@
 
 package uk.gov.hmrc.excisemovementcontrolsystemapi.config
 
-import javax.inject.{Inject, Singleton}
 import play.api.Configuration
 import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
-import scala.concurrent.duration.Duration
+import java.time.{Duration => JavaDuration}
+import java.util.concurrent.TimeUnit.MINUTES
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.duration.{DAYS, Duration, FiniteDuration, SECONDS}
 
 @Singleton
 class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig) {
 
   val appName: String = config.get[String]("appName")
 
-  def getMovementTTL: Duration = Duration(config.get[String]("mongodb.movement.TTL"))
-
   lazy val eisHost: String = servicesConfig.baseUrl("eis")
   lazy val systemApplication: String = config.get[String]("system.application")
+  lazy val interval = config.getOptional[String]("scheduler.pollingNewMessageJob.interval")
+    .map(Duration.create(_).asInstanceOf[FiniteDuration])
+    .getOrElse(FiniteDuration(5, MINUTES))
+
+  lazy  val initialDelay = config.getOptional[String]("scheduler.pollingNewMessageJob.initialDelay")
+    .map(Duration.create(_).asInstanceOf[FiniteDuration])
+    .getOrElse(FiniteDuration(60, SECONDS))
+
+  lazy val retryAfterMinutes: JavaDuration = config.getOptional[Long]("scheduler.queue.retryAfterMinutes")
+    .fold(JavaDuration.ofMinutes(5L))(JavaDuration.ofMinutes(_))
+
+  lazy val retryAttempt = config.getOptional[Int]("scheduler.queue.retryAttempt").getOrElse(3)
```

Comment:
Should this be plurarlised? i.e. `retryAttempts`

Reply:
yes. done